### PR TITLE
About: update example to use better practices

### PR DESCRIPTION
**Pull request by Fishrock123**
also for testing the latest https://github.com/nodejs-github-bot/github-bot

### CODE REVIEW

**Comment by ghost** [comment]
wouldn't no-semicolons be kind of alienating to some users?

**Comment by Fishrock123** [comment]
Shouldn't be. Since is the official documentation it should be noted that it will most definitely work, and will prepare people for the inevitable hordes of semicolonless code people will see in various modules. :)

File: locale/en/about/index.md [hunk 1]
```diff
@@ -11,17 +11,19 @@ connections can be handled concurrently. Upon each connection the callback is
...
+
+const hostname = '127.0.0.1'
+const port = 3000
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200
+  res.setHeader('Content-Type', 'text/plain')
+  res.end('Hello World\n')
+})
+server.listen(port, hostname, () => {
```
Comment (ghost) [inline locale/en/about/index.md]: can we put one blank line between this line and the previous?

Comment (Fishrock123) [inline locale/en/about/index.md]: yup

**Comment by ghost** [comment]
LGTM :)

**Comment by Fishrock123** [comment]
pardon me but um

> also for testing the latest https://github.com/nodejs-github-bot/github-bot

**Comment by Fishrock123** [comment]
nvm I'll just make a test PR then

**Comment by ghost** [comment]
@Fishrock123 wow i cant read

**Comment by lpinca** [comment]
Can we put back the semicolons? I think this is inconsistent with the API docs where semicolons are always used.

It's just an OCD of mine, don't want to start yet another semicolons discussion.

**Pull request merged**